Symfony/CS/Fixer/Symfony/AliasFunctionsFixer.php

```diff
@@ -73,7 +73,7 @@ public function fix(\SplFileInfo $file, Tokens $tokens)
             if ($prevToken->isGivenKind(array(T_NS_SEPARATOR))) {
                 $twicePrevTokenIndex = $tokens->getPrevMeaningfulToken($prevTokenIndex);
                 $twicePrevToken = $tokens[$twicePrevTokenIndex];
-                if ($twicePrevToken->isGivenKind(array(T_DOUBLE_COLON, T_NEW, T_OBJECT_OPERATOR, T_FUNCTION, T_STRING, CT_NAMESPACE_OPERATOR))) {
+                if ($twicePrevToken->isGivenKind(array(T_NEW, T_STRING, CT_NAMESPACE_OPERATOR))) {
```

Comment:
@gharlan noticed some senseless checks, so I cleaned them up in my another fixer as well.